src/reducers/splitView.js

```diff
@@ -0,0 +1,9 @@
+
+export const updateSplitPosition = (state, { value }) => {
+  localStorage.setItem('splitPosition', parseInt(value, 10))
+  return { ...state, splitPosition: value }
+}
+
+export const toggleSplitView = (state, { value }) => {
```

Comment:
There should be one reducer per file, and the filename should match the reducer name. This makes it easier find reducer definitions with some editors.

Reply:
We might end up with too many files in that case. Maybe we can couple the reducers together into the files based on their functionalities/components using them. Just a thought.  
I'll update this for now though.

Reply:
Great, we'll keep an eye on it. I haven't experienced this to be a problem yet. I find the naming convention saves a lot of time.